### PR TITLE
chore: configure Renovate manually

### PR DESCRIPTION
**Pull request by Taiki0807**
## 変更内容

- renovate 設定

### CODE REVIEW

**Comment by Taiki0807** [comment]
@HonkingGoose 
Changed title. Is this okay?

File: package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint -f stylish",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "jest --passWithNoTests",
```
Comment (HonkingGoose) [inline package.json:10]: It looks like you're ignoring the test results here? Is that really what you want?

The renovate team recommend you have _working_ tests before allowing renovate to automerge dependencies.

File: renovate.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":semanticCommits"
+  ],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "depTypeList": ["devDependencies"],
```
Comment (rarkins) [inline renovate.json:11]:
```suggestion
      "matchDepTypes": ["devDependencies"],
      "matchUpdateTypes": ["patch", "minor"],
```

File: renovate.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "automerge": true
+    }
+  ],
```
Comment (HonkingGoose) [inline renovate.json:14]:
```suggestion
```
 
If you don't have tests or are ignoring the test outcomes then you probably should remove the automerge stuff for now.

File: package.json (hunk 1 above)
Comment (Taiki0807) [reply package.json:10]: [https://github.com/Taiki0807/next.js-template/actions/runs/4745704781](https://github.com/Taiki0807/next.js-template/actions/runs/4745704781)
I am ignoring this due to a failed Github Actions run.

Comment (Taiki0807) [reply package.json]: a2e4f8f88cd514ead4c2f8166a277a78f194d777
I have a sample component ready to test, and the Github Actions come through.

File: renovate.json (hunk 2 above)
Comment (Taiki0807) [reply renovate.json]: I've made corrects

**Comment by Taiki0807** [comment]
@rarkins @HonkingGoose 
Hi!
I made some modifications. Can you please review it and provide your feedback? Appreciate your time!

File: app/components/parts/Button/Button.test.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import * as stories from './Button.stories';
+import '@testing-library/jest-dom';
+
+const { Default } = composeStories(stories)
+
+test('render Button with default args', () => {
+  render(<Default>Button</Default>);
+  const ButtonElement = screen.getByText(/Button/i);
+  expect(ButtonElement).toBeInTheDocument();
+})
```
Comment (HonkingGoose) [inline app/components/parts/Button/Button.test.tsx:12]: Needs a newline at the end of line 12.

File: app/components/parts/Button/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export {Button} from './Button';
```
Comment (HonkingGoose) [inline app/components/parts/Button/index.ts:1]: Needs a newline.

**Review by HonkingGoose — COMMENTED** [review]
The Renovate config looks OK, I did not review any of the `app/*` changes.

File: app/components/parts/Button/index.ts (hunk 5 above)
Comment (Taiki0807) [reply app/components/parts/Button/index.ts]: @HonkingGoose 
I've made corrects
Would merging this commit enable renovate to function properly?

Comment (HonkingGoose) [reply app/components/parts/Button/index.ts]: I think you're good to go on the Renovate front. 😉

Comment (Taiki0807) [reply app/components/parts/Button/index.ts]: Thank you for your support, I appreciate it.

**Pull request merged**